src/main/java/net/fabricmc/tinyremapper/TinyRemapper.java

```diff
@@ -413,7 +420,8 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 		List<ClassInstance> ret = new ArrayList<ClassInstance>();
 
 		if (file.toString().endsWith(".class")) {
-			ClassInstance res = analyze(isInput, tags, srcPath, Files.readAllBytes(file));
+			// flat class file cannot have MRJ, just set it to default version
```

Comment:
I believe they can, imagine e.g. Eclipse bin dirs that contains the equivalent of the Jar contents, which may as well be `/path/to/bin/META-INF/classes/.../Bla.class`

Reply:
I see, then I guess we need to check if the path contains `META-INF/classes` and process it.

Reply:
fixed